README.rst

```diff
@@ -81,15 +80,21 @@ setup.py:
 
     scipion3 installp -p /home/me/scipion-em-coolem --devel
 
-TIP: If installation fails, you can access pip options like:
+If installation fails, you can access pip options like:
 
 .. code-block::
 
     scipion3 python -m pip ... (list, install, uninstall)
+    
 
 **Customize it**
-    replace icon.png with your logo.
-    update the bibtex.py with your reference.
 
-**Get rid of this content and keep the readme informative**
+Replace icon.png with your logo and pdate the bibtex.py with your reference.
+
+Get rid of this content and keep the readme informative
+
+
+**Repository**
```

Comment:
Repository can be anywhere. It does not need to be here, Not even in github but gitlab or somewhere else.

We could add some links to help pages about creating a repository.